lib/moon/components/menu_item.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Moon.Components.MenuItem do
+  @moduledoc false
+  use Moon.StatelessComponent
+
+  alias Moon.Components.AsComponent
+  alias Moon.Components.MenuItem, as: MI
```

Comment:
Please do not rename. Always use full name. In our case it should be MenuItem, not MI.

Reply:
fixed